api/sheets/index.md

```diff
@@ -8,50 +8,75 @@ has_children: true
+2. Enable the Sheets add-on by clicking the Google Sheets **Extensions** menu, pointing to **Data Commons**, and then by clicking **Fill place dcids**.
+3. Use the resulting sidebar that appears on the right side of the page to start [finding DCIDs](https://docs.datacommons.org/api/sheets/get_name.html) in the United States. You can close and reopen the sidebar at any time.
 
-## Finding a place's DCID
+![](/assets/images/sheets/sheets_dc_sheets_menu.png)
+Note that none of the custom Data Commons Sheets functions (described below) will work in a given document until you have enabled the add-on by opening the **Fill place dcids** sidebar.
 
-Data Commons' Sheets add-on ships with the ability to look up a location's [DCID](/glossary.html), or unique Data Commons identifier, within the application. To use this feature, select the final destination cell of the place [DCID](/glossary.html). Go to the Sheets menu bar, and click:
 
-**"Add-ons > Data Commons > Fill place dcids"**
 
-![](/assets/images/sheets/sheets_menu_bar.png)
+## Finding a Place’s DCID
 
-A sidebar will appear on the right of the sheet, with a search bar where you can start typing the name of the place you want. From the drop down menu, select the place you want, and its [DCID](/glossary.html) will appear in the cell that you selected.
+Data Commons’ Sheets add-on ships with the ability to look up a location’s [DCID](https://docs.datacommons.org/glossary.html), within the application by using the **Fill place dcids** feature. To use this feature to find a place’s DCID:
 
-![](/assets/images/sheets/sheets_search_box.png)
+1. Select the destination cell where you want to add a place’s [DCID](https://docs.datacommons.org/glossary.html).
+2. Click in the search box for the **Fill place dcids for selected cells** section.
+3. Start typing the name of the place you are searching for. From the drop-down menu, select the place you want, and its [DCID](https://docs.datacommons.org/glossary.html) appears in the cell that you selected. For example, the following image shows the place names that match “Hawaii”.
+   ![](/assets/images/sheets/sheets_finding_dcid.png)
 
 Note that this feature only supports places in the United States.
 
-## Sorting your results
+## Introduction to the Sheets API Custom Functions
+
+The Data Commons Sheets API includes the five formulas listed in the following table. Click the links in the table for detailed information on each formula.
+
+| **Formula**                                                                                  | **Description**                           |
+|----------------------------------------------------------------------------------------------|-------------------------------------------|
+| [=DCGETNAME(_dcids_)](https://docs.datacommons.org/api/sheets/get_name.html)                 | Returns the names associated with a DCID. |
```

Comment:
```suggestion
| [=DCGETNAME(_DCIDs_)](https://docs.datacommons.org/api/sheets/get_name.html)                 | Returns the names associated with a DCID. |
```

Reply:
Here and below

Reply:
This change hasn't been made.

Reply:
I made all the dcids lowercase and removed the underscore characters to make them consistent with the rest of the pages.